crates/holochain_p2p/src/lib.rs

```diff
@@ -143,9 +143,6 @@ pub trait HolochainP2pDnaT: Send + Sync + 'static {
         message: event::CountersigningSessionNegotiationMessage,
     ) -> HolochainP2pResult<()>;
 
-    /// New data has been integrated and is ready for gossiping.
-    async fn new_integrated_data(&self) -> HolochainP2pResult<()>;
```

Comment:
deleting this... we have an issue for something like this, but it'll work differently, so we can deal with that at that point.

Reply:
Yep for sure, there's a specific issue about this -> https://github.com/holochain/holochain/issues/4675